torch/distributed/tensor/_ops/_view_ops.py

```diff
@@ -654,3 +654,45 @@ def reshape_strategy(op_schema: OpSchema) -> StrategyType:
 )
 register_op_strategy_map(aten.view_as_complex.default, torch.view_as_complex)
 register_op_strategy_map(aten.view_as_real.default, torch.view_as_real)
+
+
+@register_op_strategy(aten.as_strided.default, schema_info=RuntimeSchemaInfo(1))
+def as_strided_strategy(op_schema: OpSchema) -> StrategyType:
```

Comment:
Where do we enforce that this is only used in fake mode?

Reply:
this is the part I don't feel confident -- I did not enforce that. I only tested that it works out for my use case in Fake Tensor mode and totally rely on users to read the comment and not use it in other case.